Nos/Views/UNS/UNSNamePicker.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  UNSNamePicker.swift
+//  Nos
+//
+//  Created by Matthew Lorentz on 10/12/23.
+//
+
+import SwiftUI
+
+struct PickerRow<Label: View>: View {
```

Comment:
I prefer to declare this kind of views that are not intended to be used outside this file as fileprivate. This way we avoid a name clash in the future, and we explicitly state that PickerRow is not reusable.